common/setups/hybrid/gmm_system.py

```diff
@@ -603,12 +623,13 @@ def recognition(
         # parameters just for passing through
         corpus: str,
         feature_flow: str,
-        pronunciation_scale: float,
+        pronunciation_scale: Union[float, List[float]],
         search_parameters: dict,
         rtf: float,
         mem: float,
         parallelize_conversion: bool,
         lattice_to_ctm_kwargs: dict,
+        reestimate_prior=False,
```

Comment:
Is this function for general recognition or only GMM?

for GMM we have no prior. so no need to (re)estimate it.

Reply:
only for GMM -> removed